content/en/press-kit.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Press kit
+sidebar: false
+---
+
+We would like to make it easy for you to include the NumPy brand identity in your next academic paper, course materials or presentation. 
+
+You will find a high resolution logo of NumPy [here](https://numpy.org).
```

Comment:
For now let's link to https://github.com/numpy/numpy/tree/master/branding/icons, I don't think we have anything higher resolution. Once we have a new logo, we can update this.

Reply:
Works for me!